src/sst/elements/vanadis/os/syscall/exit.cc

```diff
@@ -44,8 +44,12 @@ VanadisExitSyscall::VanadisExitSyscall( VanadisNodeOSComponent* os, SST::Link* c
         m_buffer.resize(sizeof(uint32_t));
         *((uint32_t*)m_buffer.data()) = 0;
         writeMemory( tid_addr, m_buffer );
+        m_output->verbose(CALL_INFO, 16, VANADIS_OS_DBG_SYSCALL, "[syscall-exit] core %d thread %d process %d writeMemory() called \n",event->getCoreID(), event->getThreadID(), process->getpid() );
     } else {
+        m_output->verbose(CALL_INFO, 16, VANADIS_OS_DBG_SYSCALL, "[syscall-exit] core %d thread %d process %d memReqIsDone(true) called \n",event->getCoreID(), event->getThreadID(), process->getpid() );
+        printf("[syscall-exit] core %d thread %d process %d memReqIsDone(true) called \n",event->getCoreID(), event->getThreadID(), process->getpid());
```

Comment:
Do you need this printf? The inclusion of printf's will likely cause the autotester to fail if the outputs do not match.